ddht/tools/driver/node.py

```diff
@@ -74,3 +77,19 @@ async def network(
         async with background_trio_service(network):
             await client.wait_listening()
             yield network
+
+    @asynccontextmanager
```

Comment:
Weak preference for breaking this out into a standalone function, an `alexandria_network` context manager might be useful even if you aren't currently holding on to a `NodeAPI`. No objections to also adding a method to NodeAPI:

```python
async def alexandria(self):
  async with self.network() as network:
    async with alexandria_network(self.network) as alexandria:
       yield alexandria
```

I'm not sure what to call the standalone function though.

Reply:
This `NodeAPI` is testing infrastructure, allowing us to bundle together all the different pieces that go into a node without needing to actually run one (`enr_db`, `enr_manager`, `node_id`, `private_key`, ...).  I am reasonably confident that there isn't a standalone use case for this outside of testing.